server/services/participant-status/get-statuses.ts

```diff
@@ -0,0 +1,13 @@
+import { dbClient, collections } from '../../db';
+
+export const getParticipantHiredStatuses = async (participantId: number) => {
+  const statuses = await dbClient.db[collections.PARTICIPANTS_STATUS].find({
+    participant_id: participantId,
+    status: 'hired',
+    current: true,
```

Comment:
Should this be updated to use the ParticipantStatus const? There are a few places where the status is typed, is that an intentional decision, or were they missed? (If its intentional I could be misunderstanding the purpose of an enums const)

Reply:
That's just a result of how many literals were scattered around the codebase, but good catch for a refactor opportunity. I went through and swapped a lot of that out.